src/client/visualizations/base-visualization/base-visualization.tsx

```diff
@@ -202,10 +203,10 @@ export class BaseVisualization<S extends BaseVisualizationState> extends React.C
     this.setState({ highlight: null });
   };
 
-  protected highlight(clauses: List<FilterClause>, key: string) {
+  protected highlight = (clauses: List<FilterClause>, key: string | null = null) => {
```

Comment:
There should be some shortcut syntax for this: `string | null = null`.

Reply:
Or maybe I'm confusing runtime values with types. Is `null` on the right side of `=` a value or a type?

Reply:
It’s value. Nullable string, by default it’s null